src/routes/Calculator.js

```diff
@@ -1,8 +1,8 @@
 import { useState } from 'react';
-import CalcBtn from '../components/calcBtn';
-import CalcDisplay from '../components/calcDisplay';
-import '../css/calculator.css';
+import CalcBtn from '../components/CalcBtn'; // eslint-disable-line
+import CalcDisplay from '../components/CalcDisplay'; // eslint-disable-line
```

Comment:
- [ ] After installing dependencies and trying to start up the app, I got this error message that need to be resolved. 

### From Terminal
![start-error](https://user-images.githubusercontent.com/26597432/228853052-12d809a6-1eb8-4fa3-99fb-3c6f73215f7d.png)

### On the browser
![browser-error](https://user-images.githubusercontent.com/26597432/228853206-f29d3551-6600-4158-bb6d-97efa2507680.png)